scripts/averages.js

```diff
@@ -0,0 +1,44 @@
+var pry = require('pryjs');
+
+function total(array) {
+  var sum = array.reduce(function (total, number) {
+    return total + number;
+  }, 0);
+return sum
+};
+
+function mean(array) {
+  var total = array.reduce(function (total, number) {
```

Comment:
you should be able to call functions within other functions as long as they are defined - let me know if you have problems with this!